{"uuid":"396192c4-4a8c-4cf6-acb1-d5b93b398ac6","historyId":"8954824df3708de645d902940dfbe026","fullName":"com.nfts.Page_Test.Learn_BlogPage_Test.NavigateToLearn_BlogPage","labels":[{"name":"package","value":"com.nfts.Page_Test.Learn_BlogPage_Test"},{"name":"testClass","value":"com.nfts.Page_Test.Learn_BlogPage_Test"},{"name":"testMethod","value":"NavigateToLearn_BlogPage"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.nfts.Page_Test.Learn_BlogPage_Test"},{"name":"host","value":"DESKTOP-6K33ESR"},{"name":"thread","value":"17116@DESKTOP-6K33ESR.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"NavigateToLearn_BlogPage","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [https://www.nft.com/articles] but found [https://www.nft.com/]","trace":"java.lang.AssertionError: expected [https://www.nft.com/articles] but found [https://www.nft.com/]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1037)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:140)\r\n\tat org.testng.Assert.assertEquals(Assert.java:122)\r\n\tat org.testng.Assert.assertEquals(Assert.java:629)\r\n\tat org.testng.Assert.assertEquals(Assert.java:639)\r\n\tat com.nft.learn.learn_Blogs.NavigateToLearn_BlogPage(learn_Blogs.java:37)\r\n\tat com.nfts.Page_Test.Learn_BlogPage_Test.NavigateToLearn_BlogPage(Learn_BlogPage_Test.java:30)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\r\n\tat org.testng.TestRunner.run(TestRunner.java:596)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\r\n\tat org.testng.TestNG.run(TestNG.java:1031)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"saveFailureScreenShot","source":"bc09be6f-77c4-43c0-b27a-f3242de99465-attachment"},{"name":"NavigateToLearn_BlogPage failed and screenshot taken!","source":"8398deaf-5917-4b1f-8198-7cce71a4e61a-attachment","type":"text/plain"}],"parameters":[],"start":1681221058209,"stop":1681221068843}